ops/terraform/services/server/server-regression/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket         = "bfd-tf-state"
+    key            = "services/server/terraform.tfstate"
```

Comment:
I know our opinion of this changed a few times, but we've ultimately landed on a distinct states for `server` and `server-regression`. I'll migrate this.